tests/java/CheckstyleBearTest.py

```diff
@@ -31,3 +32,13 @@ def test_with_custom_configfile(self):
         self.section["checkstyle_configs"] = self.empty_config
         self.check_validity(self.uut, [], self.good_file)
         self.check_validity(self.uut, [], self.bad_file)
+
+
+def test_known_checkstyle(monkeypatch):
```

Comment:
monkeypatch is a fixture provided by pytest

I don't want this to be inside that TesCase since it's testing a different
unit. The import would still need to be changed since the old import gets
only the bear class, and this is testing a module-level function.

On Tue, Sep 6, 2016, 9:48 AM AbdealiJK <notifications@github.com> wrote:

> In tests/java/CheckstyleBearTest.py
> <https://github.com/coala-analyzer/coala-bears/pull/767#discussion_r77586819>
> :
>
> > @@ -31,3 +32,13 @@ def test_with_custom_configfile(self):
> >          self.section["checkstyle_configs"] = self.empty_config
> >          self.check_validity(self.uut, [], self.good_file)
> >          self.check_validity(self.uut, [], self.bad_file)
> > +
> > +
> > +def test_known_checkstyle(monkeypatch):
>
> What is monkeypatch ?
> Possibly use unittest.mock.patch ? Then it can be inside the TestCase
> class and that's what we do in other places and also you wouldn't need to
> change from bears.java.CheckstyleBear import CheckstyleBear
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/767/files/cc64ac0bc1f24a10d3a26bcdc9e8943dcf69c93a..e88fbd755e29e863c155cdbcce509d0304ac35a4#r77586819>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABAuVI2xXiKmP8_3xpMkhpfMOnRZUbHQks5qnRrYgaJpZM4JzMuc>
> .
>